frontend/src/System/Status/Donations/Donations.js

```diff
@@ -12,39 +12,39 @@ class Donations extends Component {
   render() {
     return (
       <FieldSet legend={translate('Donations')}>
-        <div className={styles.logoContainer} title="Radarr">
+        <div className={styles.logoContainer} title={translate('Radarr')}>
```

Comment:
We don't actually want people to be able to change these via translation. Thus we haven't wrapped them.